.ci/containers/terraform-vcr-tester/run_vcr_tests.sh

```diff
@@ -27,6 +27,19 @@ else
     echo "Running tests: Go files changed"
 fi
 
+
+$BRANCHNAME=$(echo -n "/auto-pr-${pr_number}"| base64)
+curl --header "Accept: application/json" --header "Authorization: Bearer $TEAMCITY_TOKEN" https://ci-oss.hashicorp.engineering/app/rest/builds/multiple/branch:name:\(\$base64:$BRANCHNAME\),buildType:id:GoogleCloudBeta_ProviderGoogleCloudBetaMmUpstreamVcr,property:\(name:env.VCR_MODE,value:REPLAYING\),defaultFilter=false --request POST --header "Content-Type:application/xml" --data-binary @/teamcitycancelparams.xml -o result.json
+
+ERRS=$(cat result.json | jq .errorCount -r)
+if [ "$ERRS" -gt "0" ]; then
+	for row in $(cat result.json | jq -r '.operationResult[] | @base64'); do
+	    build_url=$(${row} | base64 --decode | jq -r '.related.build.webUrl')
+	done
+	update_status "${build_url}" "error"
```

Comment:
This should not use update_status - clicking on the status for this PR should always get you the current build, but this build url will be for an old (broken) build.

Maybe it could be posted as a comment that includes all cancelled build URLs? (or rather - urls for builds that failed cancellation?)